src/os/bluestore/BlueFS.cc

```diff
@@ -479,6 +479,29 @@ void BlueFS::_update_logger_stats()
   }
 }
 
+void BlueFS::collect_read_stalled_alerts(osd_alert_list_t& alerts) {
+  if (!cct->_conf->bluestore_stalled_read_event_counter_threshold) {
+    return;
+  }
+  for (size_t i = 0; i < bdev.size(); ++i) {
```

Comment:
Please do collection for DB/WAL bdevs only. SLOW device is managed by BlueStore, others are just temporary devices for volume migration.